generators/app/templates/config-overrides.js

```diff
@@ -36,7 +36,25 @@ const addCamelCaseToCSSModules = (config, env) => {
   });
 };
 
+const useEslintConfig = config => {
+  config.module.rules.forEach(rule => {
+    if (rule.use) {
+      const eslintUse = rule.use.find(use => eslintLoaderMatcher(use));
+
+      if (eslintUse) {
+        eslintUse.options = {
+          ...eslintUse.options,
+          baseConfig: undefined,
+          useEslintrc: true,
```

Comment:
Is `eslintUse.options` immutable ? Do we need to copy and modify it ?

Reply:
webpack config is mutable, there's no problem in that sense. Webpack receives an object describing the configuration, rewired just captures that object before actually using it and modifies it before running the start script